### DISCUSSION TITLE
Palm registry

### DISCUSSION THREAD

**Discussion by ethanknox-palmetto** [palmetto/palm-cli#46] [Ideas]
It seems like a centralized list of palm plugins would be a good thing. Maybe this could start as a single page jekyll app or Hugo (I really like Hugo tbh) - something in its' own repo that could be hosted in GH pages and linked into? then adding plugins to the list would be simple 1-line PRs that don't require a lot of fanfare (wouldn't even need real releases, honestly) and it would encourage developers to add their plugins. 

Plus it would be a reasonable place for lots of cool early 90's retro graphics.

**Comment by jakeberesford-palmetto** [comment]
I like this idea, part of me wonders how soon we're likely to see the next plugin? A registry with a single plugin might feel a little sad

**Reply by ethanknox-palmetto** [reply]
I mean I'm about 63% done with all the commands needed for a palm-django :)

**Reply by jakeberesford-palmetto** [reply]
Exciting! I imagine you need palm multi-container done for that, too?

**Reply by ethanknox-palmetto** [reply]
yes absolutely. The ELK stack is all hand-coded with `run_on_host` at the moment

**Reply by norton120** [reply]
@jakeberesford-palmetto let me know where I can grab that multicontainer PR, I can't remember where it is (it existed before we rolled to OSS) - I'd like to pull it into my fork and see what I can do to bring it home :)

**Reply by jakeberesford-palmetto** [reply]
I set up boilerplate plugin for this, details are here: https://github.com/palmetto/palm-cli/issues/17#issuecomment-972473186

**Reply by ethanknox-palmetto** [reply]
Note to self: I need this https://github.com/palmetto/palm-cli/issues/17#issuecomment-972473186

**Reply by ethanknox-palmetto** [reply]
Speaking specifically about palm-django, one of the struggles I'm hitting is that adding elk makes it un-runnable on lower memory machines. I'm thinking about adding a `--lean` flag to the init maybe, that skips the log stack?
